tests/defaults-with-controller-configs.yml

```diff
@@ -75,3 +75,5 @@ parameters:
           cpu: 500m
       metrics:
         enabled: ${crossplane:monitoring:enabled}
+      rbacManager:
+        managementPolicy: Basic
```

Comment:
Nitpick: You don't need this in the tests as it's already specified in the component defaults that will be merged into this testcase

Reply:
you're absolutely right, I deleted it, thanks